e2e-tests/upgrade-consistency-sharded-tls/run

```diff
@@ -51,97 +51,97 @@ main() {
 	compare_generation "3" "statefulset" "${CLUSTER}-cfg"
 
 	desc 'check if service and statefulset created with expected config'
-	compare_kubectl service/${CLUSTER}-rs0 "-1150"
-	compare_kubectl service/${CLUSTER}-cfg "-1150"
-	compare_kubectl statefulset/${CLUSTER}-rs0 "-1150"
-	compare_kubectl statefulset/${CLUSTER}-cfg "-1150"
+	compare_kubectl service/${CLUSTER}-rs0 "-1160"
+	compare_kubectl service/${CLUSTER}-cfg "-1160"
+	compare_kubectl statefulset/${CLUSTER}-rs0 "-1160"
+	compare_kubectl statefulset/${CLUSTER}-cfg "-1160"
 
-	desc 'test 1.16.0'
+	desc 'test 1.16.2'
 	kubectl_bin patch psmdb "${CLUSTER}" --type=merge --patch '{
-        "spec": {"crVersion":"1.16.0"}
+        "spec": {"crVersion":"1.16.2"}
     }'
 	# Wait for at least one reconciliation
 	sleep 20
 	desc 'check if Pod started'
 	wait_cluster
-	compare_generation "4" "statefulset" "${CLUSTER}-rs0"
-	compare_generation "4" "statefulset" "${CLUSTER}-cfg"
+	compare_generation "3" "statefulset" "${CLUSTER}-rs0"
```

Comment:
Can't understand why we have 3 :(

Reply:
From what I saw - when the `crVersion` changes fro 1.16.0 to 1.16.2 the Revision does not change.. We do not update ssl so latest 1.16.0 generation == 1.16.2 1st generation

Reply:
Similarly here in the previous PR: https://github.com/percona/percona-server-mongodb-operator/blob/15c289795c751f8540e8e07027ab462f03280958/e2e-tests/upgrade-consistency-sharded-tls/run#L80  https://github.com/percona/percona-server-mongodb-operator/blob/15c289795c751f8540e8e07027ab462f03280958/e2e-tests/upgrade-consistency-sharded-tls/run#L96